test/edu-benefits/1990e/00-required.e2e.spec.js

```diff
@@ -1,51 +1,104 @@
 const E2eHelpers = require('../../e2e/e2e-helpers');
-// const Timeouts = require('../../e2e/timeouts.js');
+const Timeouts = require('../../e2e/timeouts.js');
 const EduHelpers = require('../../e2e/edu-helpers');
-// const testData = require('./schema/maximal-test.json');
+const Edu1990eHelpers = require('../../e2e/edu-1990e-helpers');
+const testData = require('./schema/maximal-test.json');
 
 module.exports = E2eHelpers.createE2eTest(
   (client) => {
-    EduHelpers.initApplicationSubmitMock('1990e');
-
     // TODO: enable once 1990e is launched
-    /*
-    // Ensure introduction page renders.
-    client
-      .url(`${E2eHelpers.baseUrl}/education/apply-for-education-benefits/application/1990e`)
-      .waitForElementVisible('body', Timeouts.normal)
-      .assert.title('Apply for education benefits: Vets.gov')
-      .waitForElementVisible('div.form-progress-buttons', Timeouts.slow)
-      .click('.form-progress-buttons .usa-button-primary');
-    E2eHelpers.overrideVetsGovApi(client);
-    E2eHelpers.overrideSmoothScrolling(client);
-    E2eHelpers.expectNavigateAwayFrom(client, '/introduction');
-
-    // Applicant information page
-    client
-      .waitForElementVisible('input[name="root_relativeFullName_first"]', Timeouts.slow);
-    EduHelpers.completeRelativeInformation(client, testData.applicantInformation.data, true);
-    client.click('.form-progress-buttons .usa-button-primary');
-    E2eHelpers.expectNavigateAwayFrom(client, '/applicant/information');
-
-    // Additional benefits page (no required fields)
-    client
-      .waitForElementVisible('label[for="root_civilianBenefitsAssistance"]', Timeouts.slow);
-    client.click('.form-progress-buttons .usa-button-primary');
-    E2eHelpers.expectNavigateAwayFrom(client, '/applicant/additional-benefits');
-
-    // Benefits eligibility page (no required fields)
-    client
-      .waitForElementVisible('label[for="root_benefit"]', Timeouts.slow);
-    client.click('.form-progress-buttons .usa-button-primary');
-    E2eHelpers.expectNavigateAwayFrom(client, '/benefits/eligibility');
-
-    // Sponsor information page
-    client
-      .waitForElementVisible('input[name="root_veteranFullName_first"]', Timeouts.slow);
-    EduHelpers.completeVeteranInformation(client, testData.sponsorVeteran.data, true);
-    client.click('.form-progress-buttons .usa-button-primary');
-    E2eHelpers.expectNavigateAwayFrom(client, '/sponsor/information');
-    */
-    client.end();
+    if (client.globals.buildtype !== 'production') {
```

Comment:
It's nitpicky, but maybe add a comment explaining this is necessary because the path isn't open when the buildtype is `'production'`. Just on the off-chance we forget this pain. I'd just hate for this to be copied over to another test that it shouldn't be, or forgotten about entirely, y'know?